code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -52,12 +52,12 @@
 /obj/projectile/bullet/rebar
 	name = "rebar"
 	icon_state = "rebar"
-	damage = 40
-	stamina = 20
+	damage = 30
+	stamina = 0
```

Comment:
Not to be nit-picky (I will be nit-picky), this var can be deleted. Same with the syndicate bullet.

Reply:
The defaults already 0, it's just a needless var is all.